client/src/components/Applications/AppForm/AppForm.js

```diff
@@ -488,6 +488,9 @@ export const AppSchemaForm = ({ app }) => {
                       .map(q => q.name)
                       .filter(
                         q =>
+                          /* normal queue on Frontera does not support 1 (or 2) node jobs and should not be listed */
+                          getSystemName(app.exec_sys.login.host) !==
+                            'Frontera' ||
                           q !== 'normal' ||
                           app.definition.parallelism !== 'SERIAL'
```

Comment:
This makes sense, and no change is really necessary, but I wonder if combining these into one expression would make it more clear that it is one particular combination of values we want to filter out
```suggestion
                          !(
                            getSystemName(app.exec_sys.login.host) ===
                              'Frontera' &&
                            q === 'normal' &&
                            app.definition.parallelism === 'SERIAL'
                          )
```

Reply:
good idea; definitely clearer. 👍

Reply:
:) @nathanfranklin  Maybe make the same change here too? https://github.com/TACC/Core-Portal/pull/446/files#diff-5c89cd6aee57ef8604aff7ab09b6f84e96f8708c0fba279b22c4a93c50eb2757R112